### ISSUE TITLE
Problem after installation: /temp/cache should be writable

### ISSUE THREAD

**Issue by lefterav** [choko/MT-ComparEval#30]
I got the following error and I fixed it manually. 
```
Unable to write to directory '../temp/cache'. Make this directory writable
```
We should either provide instructions in the readme file or make it through install.sh (which is probably not sure, since it might need root permissions ?)

**Comment by choko** [comment]
Where did you install MT-ComparEval? Did ./bin/watcher.sh produce this error?

I assume, that you installed MT-ComparEval inside /var/www and you use Apache. I think that than the application should be run with user that belongs to www-data group. Otherwise, there will be other problems.

Please, can you try the following approach? http://askubuntu.com/questions/365087/grant-a-user-permissions-on-www-data-owned-var-www

**Comment by lefterav** [comment]
Yes, this problem was made by watcher. I am using apache in a custom folder. I fixed the error manually by running "sudo chmod g+rwx temp/cache'. I am not sure, but this may be a common problem for other users, so this is why I suggested to add a clarification in the readme file.

**Comment by choko** [comment]
Yes, but I would like to propose more systematic approach. Please, can you try the approach mentioned in http://askubuntu.com/questions/365087/grant-a-user-permissions-on-www-data-owned-var-www ?
